build.gradle

```
@@ -83,18 +83,18 @@ repositories {
 // Spotless checks will be added as PRs are applied to resolve each style issue is approved.
 spotless {
     java {
-//        target fileTree('.') {
-//            include '**/*.java', 'src/*/java/**/*.java'
-//            exclude '**/build/**', '**/build-*/**'
-//        }
+        target fileTree('.') {
+            include '**/*.java', 'src/*/java/**/*.java'
```

We don't have `src` in the repo root, I think one condition is enough
```suggestion
            include '**/*.java'
```

---

But we can cover `json` files there

---

Thanks, removed